PublicMapping/districtbuilder
# Add favicons

## Overview

Just adding some favicons over here.

### Demo

<img width="346" alt="Screen Shot 2020-09-01 at 11 02 23 PM" src="https://user-images.githubusercontent.com/1809908/91976434-f0876c80-ecee-11ea-8de6-7b8c22c3ecfb.png">
<img width="169" alt="Screen Shot 2020-09-01 at 11 02 30 PM" src="https://user-images.githubusercontent.com/1809908/91976436-f0876c80-ecee-11ea-8a98-a9e5872c65f5.png">

## Testing Instructions

- Open the application
- Do you see favicons in places you'd expect to see favicons?

Closes #272
